src/internals/domain.js

```diff
@@ -82,7 +82,8 @@ extend(ChartInternal.prototype, {
 		let yDomainMax = $$.getYDomainMax(yTargets);
 
 		const center = config[`axis_${axisId}_center`];
-		let isZeroBased = ($$.hasType("bar", yTargets) && config.bar_zerobased) || ($$.hasType("area", yTargets) && config.area_zerobased);
+		let isZeroBased = ($$.hasType("bar", yTargets) && config.bar_zerobased) || ($$.hasType("area", yTargets) && config.area_zerobased) ||
+						($$.hasType("line", yTargets) && config.line_zerobased) || ($$.hasType("scatter", yTargets) && config.scatter_zerobased);
```

Comment:
### 1) Add for bubble type

setting `zerobased` is somehow equivalent of `axis.y.min=0` and it can be applied for all Axis based types, which also include `bubble` type.

So, adding `bubble.zerobased` option also needed here.

### 2) Get rid duplication
As of the increase on type checking, there're duplication here. Simplifying as will be more efficient.

```js
let isZeroBased = ["area", "bar", "bubble", "line", "scatter"]
    .some(v => $$.hasType(v, yTargets) && config[`${v}_zerobased`]);
```